@navikt/core/react/src/modal/ModalUtils.ts

```diff
@@ -18,8 +18,10 @@ export const BODY_CLASS = "navds-modal__document-body";
 export function useBodyScrollLock(
   modalRef: React.RefObject<HTMLDialogElement>,
   portalNode: HTMLElement | null,
+  isNested: boolean,
 ) {
   React.useEffect(() => {
+    if (isNested) return;
```

Comment:
React Hook React.useEffect has a missing dependency: 'isNested'. Either include it or remove the dependency array

Reply:
Bra spottet. Var noe bug med vscode som gjorde at jeg ikke fikk opp eslint-feil.